_includes/moto.md

```diff
@@ -0,0 +1,92 @@
+{% assign api_resource=include.api_resource %}
+{% if include.api_resource == "creditcard" %}
+    {% assign api_resource_field_name = "payment" %}
+{% else %}
+    {% assign api_resource_field_name = "paymentorder" %}
+{% endif %}
+
+## MOTO (Mail Order / Telephone Order)
+
+{% include alert-agreement-required.md %}
+
+MOTO (Mail Order / Telephone Order) is a purchase where you as a merchant enter 
+the customer card details in order to make a payment. The card details are given
+over telephone or in writing, and then entered into the payment interface by the
+sales representative. Common use-cases are travel or hotel bookings, where the 
+customer calls the sales representative to make a booking. This feature is only 
+supported in the `Purchase` operation and is only valid if you have 3-D Secure 2
+authentication of the cardholder. See the abbreviated example below on how to
```

Comment:
This might be misleading. Moto is only available if you have 3-D Secure 2 authentication activated in the contract. The purchase will never prompt a Strong Customer Authentication on the Cardholder for MOTO (as the cardholder is not present).